pipeline_tools/dcp_utils.py

```diff
@@ -28,8 +28,22 @@ def get_file_by_uuid(file_id, dss_url, http_requests):
 
 def get_manifest(bundle_uuid, bundle_version, dss_url, http_requests):
     """Retrieve manifest JSON file for a given bundle uuid and version.
+
     Retry with exponentially increasing wait times between requests if there are any failures.
 
+    TODO: Reduce the number of lines of code by switching to use DSS Python API client.
```

Comment:
Can you please make a ticket for this?

Reply:
Also can you fix the note about the return value? I know we'll replace this function soon, but I found this confusing! https://github.com/HumanCellAtlas/pipeline-tools/blob/9585c5c3a696bbdac94bdb070bbc2222b97bc0e0/pipeline_tools/dcp_utils.py#L53

Reply:
Good catch, I will fix the incorrect docstring here. Also, will make a ticket for this!